### ISSUE TITLE
File system error on Windows

### ISSUE THREAD

**Issue by carstenhag** [kirillrogovoy/pullkee#1]
Hello, I've tried to run `pullkee --limit 20 facebook/react` on Windows 10 x64 but sadly Pullkee is giving out many errors about not being able to find the path:

```
2017/09/28 21:49:04 File system error occurred while accessing the cache: open C:\Users\carstenh\AppData\Local\Temp/pullkee_cache/facebook/react/pr10019.json: Das System kann den angegebenen                                                      Pfad nicht finden.
2017/09/28 21:49:04 File system error occurred while accessing the cache: open C:\Users\carstenh\AppData\Local\Temp/pullkee_cache/facebook/react/pr10018.json: Das System kann den angegebenen                                                      Pfad nicht finden.
2017/09/28 21:49:04 File system error occurred while accessing the cache: open C:\Users\carstenh\AppData\Local\Temp/pullkee_cache/facebook/react/pr10025.json: Das System kann den angegebenen                                                      Pfad nicht finden.
2017/09/28 21:49:04 File system error occurred while accessing the cache: open C:\Users\carstenh\AppData\Local\Temp/pullkee_cache/facebook/react/pr10028.json: Das System kann den angegebenen                                                      Pfad nicht finden.

```

"Das System kann den angegebenen Pfad nicht finden" translates to "The system could not find the specified path".

It probably has something to do with forward slashes being used instead of backslashes?

**Comment by kirillrogovoy** [comment]
@carstenhag Thank you very much for bringing this up!

I believe, this issue has gotten fixed in #2.

I've made a new release - 1.2.1, where it is fixed.

Let me know if you have any further issues.
Please, close the issue if everything works well.

**Comment by kirillrogovoy** [comment]
Closing this issue for inactivity. Let me know if this problem persists for you in the newer releases.
